### DISCUSSION TITLE
Apply for github sponsors?

### DISCUSSION THREAD

**Discussion by Ainali** [govdirectory/website#496] [Ideas]
I got a notification that this organization is eligible for GitHub Sponsors. Perhaps we should apply when we have something viewable?

**Comment by Abbe98** [comment]
Related link: https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-organization

**Comment by Ainali** [comment]
Related question (but should perhaps be split to it's own issue): Should we start a collective on https://opencollective.com/ to be able to receive funding in various ways?  If we choose a Fiscal host there, we could then receive money without having to incorporate an organization, making it more lightweight.

**Comment by Abbe98** [comment]
Maybe both of these are a part of / dependent upon a bigger discussion regarding sustainability which we should tackle in September.

**Comment by pnarayanan01** [comment]
Not sure if you guys went ahead and decided not to do this, but in case you have not yet decided, I think it is a good idea to apply for it. 
I am not even sure if this is the right place to comment on it, so forgive my trespassing.
